lib/core/models/dummy_data.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:palakat/core/models/models.dart';
+
+class DummyData {
+  List<String> titles = [
+    'KJ NO.999',
+    'KAMI PUJI DENGAN RIANG DIKAY ALLAH YANG BESAR',
+  ];
+  List<String> source = [
```

Comment:
ini boleh gabung dalam List<SongPart>, biar nanti looping di satu tampa drng

Reply:
ini si variable source boleh tambah jo di variable List<SongPart> data. Biar nanti pas looping di screen satu kali samua dia. ndpp tamba property jo di SongPart khusus for variable source itu